recipes/pydoe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: pydoe
+  version: {{ version }}
+
+source:
+    fn: pyDOE-afc5949f01c010a26764adba9624871d51797270.tar.gz
+    url: https://github.com/tisimst/pyDOE/archive/afc5949f01c010a26764adba9624871d51797270.tar.gz
```

Comment:
Yes. I can ask. Is this blocking? I can update the recipe if they add a git-tag later.